src/battle-scene.ts

```diff
@@ -262,18 +202,8 @@ export default class BattleScene extends SceneBase {
   public gameData: GameData;
   public sessionSlotId: number;
 
-  /** PhaseQueue: dequeue/remove the first element to get the next phase */
-  public phaseQueue: Phase[];
-  public conditionalQueue: Array<[() => boolean, Phase]>;
-  /** PhaseQueuePrepend: is a temp storage of what will be added to PhaseQueue */
-  private phaseQueuePrepend: Phase[];
+  public phaseManager: PhaseManager;
```

Comment:
NIT: I assume this doesn't change after init anymore? Should be `readonly` then to prevent modification from outside